test/capability-tool-test.js

```diff
@@ -0,0 +1,201 @@
+/* eslint-disable func-names, no-unused-expressions */
+const fs = require('fs');
+const path = require('path');
+
+const { expect } = require('chai');
+
+const CapabilityTool = require('../lib/capability-tool');
+
+function readFixture(filename) {
+  return JSON.parse(fs.readFileSync(path.normalize(`${__dirname}/fixtures/${filename}`, 'utf8')));
+}
+
+describe('CapabilityTool', function () {
+  beforeEach(function () {
+    const capabilityStatement = readFixture(('valid-capability-statement.json'));
+    this.capabilities = new CapabilityTool(capabilityStatement);
+  });
+
+  describe('#serverCan', () => {
```

Comment:
We shouldn't use fat-arrow in our specs as a matter of style/convention, see: https://mochajs.org/#arrow-functions